### PR TITLE
1053 lyd ebog filter stvi

### PR DESCRIPTION
**Pull request by stanvins**

### CODE REVIEW

**Comment by stanvins** [comment]
@kfudbc - Not sure why this integration test is failing. Seemed to fail last time I merged something as well.

File: src/client/components/filter/FilterPage/ResultsFilter.component.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      }
+      break;
+    case ALLBOOKS:
+      retType = type;
+      break;
+    default:
+      retType = type;
+      break;
+  }
+  changeType(retType);
```
Comment (kfudbc) [inline src/client/components/filter/FilterPage/ResultsFilter.component.js]: Umiddelbart vil jeg ikke mene at en reducer-funktion bør kalde en "callback"-funktion

File: src/client/redux/recommend.thunk.js [hunk 2]
```diff
@@ -171,12 +171,14 @@ const fetchRecommendations = async action => {
     }, {});
 
     // thresholdLevel = Math.floor(Math.sqrt(Object.keys(tagsMap).length));  // Disabled until further investigated - see US1058
-
     query.tags = tagsMap;
     query.plus = action.plusArr;
     query.minus = action.minusArr;
     query.maxresults = limit;
     query.creators = creators;
+    if (action.types && query.types !== ['Bog']) {
```
[the remark this answers is not in the document]
Comment (stanvins) [reply src/client/redux/recommend.thunk.js]: This was a left over check that no longer makes sense, so I removed it.

**Comment by stanvins** [comment]
I have put this same code up 3 times now. It seems to pass and fail the integration testing for random reasons. When it fails I cant get it to repeat them locally on cypress.

File: src/client/components/filter/FilterPage/ResultsFilter.component.js (hunk 1 above)
Comment (stanvins) [reply src/client/components/filter/FilterPage/ResultsFilter.component.js]: I was trying to find an excuse to use hooks. This was based on a template and it got unnecessarily complex. I have changed it to a normal component.

**Pull request merged**